.github/workflows/ci.yml

```diff
@@ -59,6 +59,8 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: npm
+    - name: Update npm
+      run: npm install -g npm@8
```

Comment:
For our Node 14 tests we need to make sure we're using `npm` v8 to support the "workspaces" feature in package.json

Reply:
Does npm 9 still run tests on 14?

Reply:
ah yeah looks like it https://github.com/npm/cli/blob/latest/.github/workflows/ci.yml#L70-L71